pkgs/by-name/ph/photini/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "photini";
+  version = "2024.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jim-easterbrook";
+    repo = "Photini";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-CYuMMySCQ2d0aVqfXHcRjXnH5H59bPtueCKqjso7QmU=";
+  };
+
+  build-system = with python3Packages;  [
+    setuptools-scm
+    toml
```

Comment:
```suggestion
```
Iirc the toml feature is automatically enabled and adding that here and upstream is redundant

Reply:
Yes, seems to work without toml here. Weird, I thought I checked it before and it didn't work without it.